CHANGELOG.md

```diff
@@ -16,6 +16,11 @@
   ([**@danielfromearth**](https://github.com/danielfromearth),[**@chuckwondo**](https://github.com/chuckwondo),
    [**@jhkennedy**](https://github.com/jhkennedy),[**@mfisher87**](https://github.com/mfisher87))
 
+- Deprecate `earthaccess.get_s3fs_session` and `Store.get_s3fs_session`. Use
+  `earthaccess.get_s3_filesystem` and `Store.get_s3_filesystem`, respectively,
+  instead ([#766](https://github.com/nsidc/earthaccess/issues/766))
```

Comment:
Since we just did a release, this should be alone in an "unreleased" section. Do you feel comfortable rebasing this branch to pull in newly-merged changes?

Also, please follow the [common changelog](https://common-changelog.org/) format and add yourself as the author of this change ;)

Reply:
Hey Matt, I have already pulled in the new changes made to CHANGELOG.md. I guess rebasing is not needed in here we can manually change this commit (amend the changes to the same commit).

Reply:
@Sherwin-14, I think this is something we should cover in today's hacking session, as you have not pulled in the changes that Matt and I have referred you to pull in. We can show you how to pull changes from upstream and rebase your branch so that you are up to date with what's on the upstream main branch.